meta-resin-common/recipes-support/resin-mounts/resin-mounts/resin-partition-mounter

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+set -e
+
+help () {
+	cat << EOF
+Mount a resin partition label
+$0 <ARGUMENTS>
+
+ARGUMENTS:
+	-h, --help
+		Print this message.
+	--mount <LABEL>
+		Action: Mount label
+	--umount <LABEL>
+		Action: Unmount label
+EOF
+}
+
+# Parse arguments
+while [ "$#" -ge 1 ]; do
+	i="$1"
+	case $i in
+		-h|--help)
+			help
+			exit 0
+			;;
+		--mount)
+			if [ -z "$2" ]; then
+				echo "ERROR: $1 argument needs a value."
+			fi
+			part_label="$2"
+			action="mount"
+			shift
+			;;
+		--umount)
+			if [ -z "$2" ]; then
+				echo "ERROR: $1 argument needs a value."
+			fi
+			part_label="$2"
+			action="umount"
+			shift
```

Comment:
This is not needed as I don't think you intend to have your script called with more than 2 args at once

Reply:
Sure but the script might change so it's good to code thinking a little about the future.